posts/views.py

```diff
@@ -428,6 +437,120 @@ def get(self, request, authorid):
                 contentTypes.append( "<p>" + post.content +"</p>")
         return render(request, 'author/author_posts.html', context={'author': author, 'posts':serializer.data, 'contentTypes':contentTypes})
 
+class GetAuthorPosts(views.APIView):
```

Comment:
I think this should be moved to the feed view file just because it has to do with the author feeds.